web/sites/default/config/views.view.content.yml

```diff
@@ -1143,15 +1143,24 @@ display:
           admin_label: 'Content group content'
           required: false
           group_content_plugins:
-            'group_node:city_service': 'group_node:city_service'
-            'group_node:event': 'group_node:event'
-            'group_node:guide': 'group_node:guide'
-            'group_node:information': 'group_node:information'
-            'group_node:news': 'group_node:news'
-            'group_node:notification': 'group_node:notification'
-            'group_node:page': 'group_node:page'
             'group_node:alert': '0'
             'group_node:building': '0'
+            'group_node:calendar_item': '0'
+            'group_node:city_charter': '0'
+            'group_node:city_code': '0'
```

Comment:
Resave this view.

Reply:
Group has been re-saved and no changes were shown. These items are expected here, since the relationship is set to select all content types, same as in POWR-1544.

Reply:
I resaved the view, but there were no changes. This is expected because we changed the group content relationship to use all content types. It lists them all here even if they group content plugins aren't enabled in any group type. This is same as my comment in 1544.